src/context.ts

```diff
@@ -42,7 +44,7 @@ export class OperonContextImpl implements OperonContext {
   workflowUUID: string = "";          // Workflow UUID. Empty for HandlerContexts.
   readonly logger: OperonLogger;      // Wrapper around the global logger for this context.
 
-  constructor(readonly operationName: string, readonly span: Span, logger: Logger, parentCtx?: OperonContextImpl) {
+  constructor(readonly operationName: string, readonly span: Span, logger: Logger, readonly applicationVersion: string, parentCtx?: OperonContextImpl) {
```

Comment:
Asking for `applicationVersion` in `OperonContext` allows each context's logger to access the application version and inject it in log entries